src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FlowExtended.groovy

```diff
@@ -426,6 +444,14 @@ class FlowExtended {
         northbound.deleteFlow(flowId)
     }
 
+    /**
+     * This method gets info about connected devices to flow src and dst switches
+     * @param since can be used to get details about connected devices starting from a specific time
+     */
+    FlowConnectedDevicesResponse retrieveConnectedDetails(String since = null) {
```

Comment:
It's always better use Time (e.g. DateTime or yoda time) classes to represent time. Please, consider it (or at least adding TODO).

Reply:
For now, it seems like an unnecessary transformation for the timeLastSeen from `ConnectedDeviceDto` as we need to parse String to Date and then again to String as northbound.getFlowConnectedDevices also uses String.

Btw, we can modify it if it is required in the future OR we can add our own '`ConnectedDeviceDto`' to parse timestamp properly.